torch/csrc/jit/test_jit.cpp

```diff
@@ -911,6 +913,7 @@ std::string runJITCPPTests() {
   fromQualStringTests();
   argumentSpecTest();
   shapeAnalysisTest();
+  */
   testProto();
```

Comment:
Since `testProto()` doesn't take `out` as argument, do we expect `out.str()` to return anything in this case? Just trying to understand how the test case is expected to work in this scenario.

Reply:
On windows the call to `proto.DebugString()` will thrown an exception. It appears to be related to the way protobufs are built. This PR is to repro the issue separate from the PR that removes nano_pb in favor of protobufs.